Resources/ArmTemplates/test-parameters.json

```diff
@@ -40,9 +40,9 @@
         "sharedAppServicePlanResourceGroup": {
             "value": "dss-at-shared-rg"
         },
-        "templateBaseUrl": {
-            "value": "https://raw.githubusercontent.com/SkillsFundingAgency/dss-contentpushservice/NCSLT-5169-v2/Resources/ArmTemplates"
-        },
+      "templateBaseUrl": {
+        "value": "https://raw.githubusercontent.com/SkillsFundingAgency/dss-contentpushservice/AD-185409-v3/Resources/ArmTemplates"
```

Comment:
This is concerning - what does this do and why is the branch name hardcoded?
CC @ajones-DFE @mimran-DFE @thomas-petty-dfe

Reply:
It was probably set that way for testing purposes and never set reverted and accidentally merged.

I would argue that it should be master at all times other than for testing.

Reply:
Resolved this by pointing it to master